jellyfin-core/src/main/kotlin/org/jellyfin/sdk/discovery/RecommendedServerDiscovery.kt

```diff
@@ -111,27 +111,27 @@ public class RecommendedServerDiscovery(
 		)
 		val api = SystemApi(client)
 
-		val info: Result<Response<PublicSystemInfo>>
-		val responseTime = measureTimeMillis {
```

Comment:
[That method](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.system/measure-time-millis.html) is not common??

Reply:
Only jvm and native